packages/slice-machine/src/features/sync/AutoSyncProvider.tsx

```diff
@@ -110,7 +111,7 @@ export const AutoSyncProvider: FC<PropsWithChildren> = (props) => {
 
           // Hard limit reached is the only limit that can be reached as we set
           // confirmDeleteDocuments to true.
-          if (response && response.type === "HARD") {
+          if (response && response.type === PushChangesLimitType.HARD) {
```

Comment:
⚠️ Let's use string literals instead of referencing the constants. For TypeScript there's no difference, and for us it's less code to maintain (you don't have to import and reference and can use just the value directly).

Reply:
Alright, I guess the real fix is to stop using enums and use unions instead. But we can do this incrementally when we see the opportunity. For legacy enums, I prefer to reference them instead of disabling the rule (just because it's safer), I didn't realize this one was an enum. 

But you can also leave it like this if exporting the enum is causing problems.